google/cloud/pubsub/internal/streaming_subscription_batch_source.h

```diff
@@ -167,7 +170,9 @@ class StreamingSubscriptionBatchSource
   bool pending_read_ = false;
   Status status_;
   std::shared_ptr<AsyncPullStream> stream_;
+  bool exactly_once_delivery_enabled_ = false;
```

Comment:
This one, I thought, needed to stay as `optional` to catch the transition from unengaged to false.  But I'm surmising.